db/src/db.rs

```diff
@@ -995,6 +1009,57 @@ pub fn update_partition_map(conn: &rusqlite::Connection, partition_map: &Partiti
     Ok(())
 }
 
+fn metadata_assertions(conn: &rusqlite::Connection, tx_id: Entid, comitted: bool) -> Result<Vec<(Entid, Entid, TypedValue, bool)>> {
```

Comment:
This isn't a function with a `bool` parameter, it's two functions.

Reply:
Splitting functions around queries (where db operation is a common bit) makes more sense than the current approach, I think.